test/libsolidity/util/SoltestTypes.h

```diff
@@ -286,12 +293,17 @@ struct FunctionCall
 		/// Marks a library deployment call.
 		Library,
 		/// Check that the storage of the current contract is empty or non-empty.
-		Storage
+		Storage,
```

Comment:
Can we later change this to be a builtin?

Reply:
Sure!